catalogData/redis32/service.json

```diff
@@ -1,12 +1,12 @@
 {
     "id": "9f54be76-722b-11e7-ad8f-6fd2bc5f79fe",
     "name": "redis32",
-    "description": "An open source in-memory data structure store.",
+    "description": "An open source in-memory data structure store with Redis Sentinel for high-availability.",
     "metadata":{
-      "displayName": "Redis 3.2",
+      "displayName": "Redis 3.2.10",
```

Comment:
I would probably leave this as 3.2 so we can upgrade minor versions for security, etc.